src/system-script.js

```diff
@@ -25,13 +26,9 @@ function normalizeName(name) {
 }
 
 function getScript(address, name) {
-  const existingScripts = Array.prototype.filter.call(
-    document.querySelectorAll("script"),
-    script => script.src === address
-  );
-
-  if (existingScripts.length) {
-    return existingScripts[0];
+  const existingScript = document.querySelector(`script[src="${address}"]`);
```

Comment:
A small performance enhancement that I tried to implement in #7 but somehow messed up with some merge conflict resolution shenanigans. So this code here is doing what #7 failed to do